rusk-wallet/src/gql.rs

```diff
@@ -20,6 +24,17 @@ pub struct GraphQL {
     status: fn(&str),
 }
 
+/// The tx_for_block returns a Vec<TransactionBlock> which contains
+/// the execution-core transaction, its id hash and gas spent
+pub struct TransactionBlock {
+    /// The execution transacton struct obtianed from grahpql endpoint
```

Comment:
```suggestion
    /// The execution transaction struct obtained from the GraphQL endpoint
```

I'm not sure what an execution transaction is though?